test/scripts/e2e_subs/e2e-app-cross-round.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+date '+app-stateful-global-test start %Y%m%d_%H%M%S'
+
+set -e
+set -x
+set -o pipefail
+export SHELLOPTS
+
+WALLET=$1
+
+# Directory of this bash program
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+
+gcmd="goal -w ${WALLET}"
+
+ACCOUNT=$(${gcmd} account list|awk '{ print $3 }')
+
+# Succeed in creating app that approves transactions with arg[0] == 'hello'
+printf '#pragma version 2\nint 1' > "${TEMPDIR}/int1.teal"
+APPID=$(${gcmd} app create --creator ${ACCOUNT} --approval-prog ${DIR}/tealprogs/cross-round.teal --global-byteslices 2 --global-ints 2 --local-byteslices 2 --local-ints 2 --clear-prog "${TEMPDIR}/int1.teal" | grep Created | awk '{ print $6 }')
```

Comment:
Maybe you should change the schema limits from 2 to 1 each? The program stores only one of each, so this could test whether reading/deleting an unknown key wrongly increases the schema count.